### PR TITLE
Adding permissions to the bulk user creation endpoint

### PR DESCRIPTION
**Pull request by igalarzab**

### CODE REVIEW

File: src/presentation/api/AdminController.test.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  beforeEach(async () => {
+    await cleanupDatabase();
+  });
+
+  describe('POST /users', () => {
+    it('successfully creates new users', async () => {
+      const authenticatedUser = await persistedUserWithRoleAndPermissions(ADMIN, [BULK_CREATE_USERS]);
+
+      await request(app)
+        .post(`/api/v1/admin/users`)
```
Comment (kstam) [inline src/presentation/api/AdminController.test.ts:22]: shouldn't this be `bulk-users` ?

Comment (kstam) [reply src/presentation/api/AdminController.test.ts:22]: Hmm.. Ignore me.. You prefixed it with `/admin`. Makes sense 👍

Comment (igalarzab) [reply src/presentation/api/AdminController.test.ts:22]: As it's inside of the /admin prefix, I think a POST to /users is enough (as the only reason to create users behing the admin endpoint is to do it bulk, even if you only create one). But I don't have strong preferences... `bulk-users` doesn't sound very RESTfully, that's why I didn't use it in a first instance

Comment (oliverviljamaa) [reply src/presentation/api/AdminController.test.ts:22]: Under `/admin`, I prefer just `users` as well 👍

**Pull request merged**